bson/bson.go

```diff
@@ -352,6 +352,50 @@ func Now() time.Time {
 // strange reason has its own datatype defined in BSON.
 type MongoTimestamp int64
 
+// Time returns the time part of ts which is stored with second precision.
+func (ts MongoTimestamp) Time() time.Time {
+	b := make([]byte, 8, 8)
+	binary.BigEndian.PutUint64(b, uint64(ts))
+	return time.Unix(int64(binary.BigEndian.Uint32(b[:4])), 0)
```

Comment:
Of course. Doh! On second thought in general, I am not sure wether _not_ supporting dates before epoch is a smart choice the first place. It might well be that the type will be used for applications outside the oplog. What are your thoughts on that?